TSPL.docc/GuidedTour/GuidedTour.md

```diff
@@ -25,8 +25,7 @@ print("Hello, world!")
   ```
 -->
 
-If you have written code in C or Objective-C,
-this syntax looks familiar to you ---
+This syntax should look familiar if you know another language---
 in Swift, this line of code is a complete program.
 You don't need to import a separate library for functionality like
 input/output or string handling.
```

Comment:
@amartini51 Alex or others to review, this wasn't part of my marketing update.

Reply:
@chuckdude Was the goal of the edit to avoid the slash, to avoid using "input" and "output" as a noun, or something else?  In Swift, input and output aren't statements.  That's true of many other languages that folks might be mentally contrasting it with, including the C family, Python, and Ruby — all of which have I/O functions.

To show the contrast, in many C-like languages you need to explicitly import a library like `stdio` or `iostream` or `io` at the start of the program before you can print output.  In Swift, `print(...)` is part of the standard library, along with many other commonly used APIs — and they're all available without any boilerplate.

Reply:
Took a rewrite pass in 4795332f.